src/components/AutoAdvanceToggle/AutoAdvanceToggle.test.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { fireEvent, render, screen } from '@testing-library/react';
+import AutoAdvanceToggle from "./AutoAdvanceToggle";
+import manifest from '@TestData/lunchroom-manners';
+import autoAdvanceManifest from '@TestData/multiple-canvas-auto-advance';
+import { withManifestProvider } from '../../services/testing-helpers';
+import * as utils from '@Services/utility-helpers';
+
+describe('AutoAdvanceToggle', () => {
+  describe('with manifest without auto-advance behavior', () => {
+    beforeEach(() => {
+      const NonAutoManifest = withManifestProvider(AutoAdvanceToggle, {
+        initialState: { manifest, canvasIndex: 0 },
+      });
+      render(<NonAutoManifest />);
+    });
+    test('renders auto advance toggle successfully', () => {
+      expect(screen.queryByTestId('auto-advance')).toBeInTheDocument();
+      expect(screen.queryByTestId('auto-advance-label')).toBeInTheDocument();
+      expect(screen.queryByTestId('auto-advance-toggle')).toBeInTheDocument();
+    });
+    test('auto advance is disabled by defulat', () => {
+      const toggle = screen.queryByTestId('auto-advance-toggle');
+      expect(toggle.checked).toEqual(false);
+    });
+    test('auto advance is toggled on click', () => {
+      const toggle = screen.queryByTestId('auto-advance-toggle');
+      fireEvent.click(toggle);
+      expect(toggle.checked).toEqual(true);
+    });
+  });
+  describe('with manifest with auto-advance behavior', () => {
+    beforeEach(() => {
+      const AutoAdvanceWithManifest = withManifestProvider(AutoAdvanceToggle, {
```

Comment:
This should probably be renamed to `AutoManifest` or `AutoAdvanceManifest`. Unsure what my original logic with the name was, but this naming is inaccurate and potentially confusing.